examples/benchmark.py

```diff
@@ -67,9 +67,9 @@
     devito.add_argument("-a", "--autotune", action="store_true",
                         help=("Benchmark with auto tuning on and off. " +
                               "Enables auto tuning when execmode is run"))
-    devito.add_argument("-cb", "--cache_blocking", nargs=2, type=int,
-                        default=None, metavar=("blockDim1", "blockDim2"),
-                        help="User provided block sizes when auto-tuning is off")
+    # devito.add_argument("-cb", "--cache_blocking", nargs=2, type=int,
```

Comment:
I saw that and thought the merge between our PRs would be easier this way. I needed to remove it as well since `arguments` won't silently ignore extra arguments now - it raises Exceptions.